src/tests/add_remove_li.test.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable import/named */
+/**      A function for editing the task description.       */
+/* eslint-disable quotes */
+import { addTask, removeTask } from "./tests-files/add_remove.js";
+import { displayTasks, tasks } from "./tests-files/index.js";
+
+jest.mock("./tests-files/add_remove");
+
+displayTasks();
+
+describe("Test task manager ->", () => {
+  test("add 1 li element to the ul", () => {
+    const input = global.document.getElementById("description");
+    input.value = "Do stuff";
+    // const task = {
+    //   id: 4, index: 4, description: "Do stuff", completed: true,
+    // };
```

Comment:
Please remove the commented-out code blocks. They make your project look unprofessional.

Reply:
Thank you for pointing that out it will be sorted out with immediate effect